src/main/java/no/nav/fo/veilarbregistrering/oppgave/OppgaveType.java

```diff
@@ -1,5 +1,31 @@
 package no.nav.fo.veilarbregistrering.oppgave;
 
-public enum OppgaveType {
-    OPPHOLDSTILLATELSE
+import no.nav.fo.veilarbregistrering.metrics.Metric;
+
+public class OppgaveType implements Metric {
```

Comment:
Trenger du en egen type for Oppgavetype i tillegg til enum Type? Kan ikke bare Oppgavetype være en enum og implementere Metrics.

Reply:
Kanskje? Kan et enum implementere et interface? 🤔